Source/Internal/zip_util.cpp

```diff
@@ -113,10 +113,11 @@ void ZipFile::AddFile(const std::string& src_file_path,
     }
 
     {
-        FILE* file;
+        FILE* file = nullptr;
         file = FOPEN_FUNC(src_file_path.c_str(), "rb");
         if (file == NULL) {
             FatalError("Error", "Could not open file: %s", src_file_path.c_str());
+            return;
```

Comment:
does FatalError call exit? if not, other FatalError calls below may also need an early return.

Reply:
FatalError does exit; this was an experiment to see if I could remove some compiler warnings and why they were being issued in the first place. This can be removed and functionality should not change.